fabric/.mcdev.template.json

```diff
@@ -222,6 +231,18 @@
       "condition": "$LANGUAGE == 'Kotlin' && $MOD_ENVIRONMENT != 'server'",
       "openInEditor": true
     },
+    {
+      "template": "DataGenerator.java.ft",
+      "destination": "src/${CLIENT_SOURCE_SET}/java/${MAIN_CLASS.packagePath}/client/${MAIN_CLASS.className}DataGenerator.java",
+      "condition": "$USE_DATAGEN",
+      "openInEditor": true
+    },
+    {
+      "template": "DataGenerator.kt.ft",
+      "destination": "src/${CLIENT_SOURCE_SET}/kotlin/${MAIN_CLASS.packagePath}/client/${MAIN_CLASS.className}DataGenerator.kt",
+      "condition": "$USE_DATAGEN",
+      "openInEditor": true
+    },
```

Comment:
I don't think we should be generating both a Java file and a Kotlin file. If anything, there should be a choice between the two. However, given the lack of support for Kotlin in the rest of the generator, I think Kotlin should be left out for now, until it is added to everything else.

Reply:
> the fabric template already has kotlin support

Ah, I didn't realize. This is fine then with the added condition